source/templates.tex

```diff
@@ -1375,11 +1377,7 @@
 \tcode{float},
 and list of pointers to
 \tcode{Shape}s.
```

Comment:
This sentence seems strange; it appears to be suggesting that a `List` template might only work for some finite set of specified types. Maybe we could remove this (or rephrase it) rather than incorporating it into the somewhat-more-reasonable `Array` example.